app/controllers/application_controller.rb

```diff
@@ -4,6 +4,10 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
 
+  def default_url_options(options={})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `default_url_options(*)` if you want the method to accept any arguments but don't care about them.<br>Surrounding space missing in default value assignment.